app/assets/javascripts/comments.js

```diff
@@ -50,6 +50,14 @@ $(() => {
     // no need to do anything else at this point in this block, until proven otherwise.
   });
 
+  $('.comment-form').find('input[value="Nope"]').on('click', function (data) {
+      const $input_button = $(data.target);
+      const $div_actions = $input_button.parent();
+      const $div_form_group_horizontal = $div_actions.parent();
+      const $form_js_comment_form = $div_form_group_horizontal.parent();
+      $form_js_comment_form.hide();
+  });
```

Comment:
* Variable names should be camelCase in JavaScript
* Also, changes for updated button HTML and for simplicity:

```suggestion
  $('.comment-form').find('input[name="cancel-comment"]').on('click', function(data) {
      const $tgt = $(data.target);
      const $form = $tgt.parents('form');
      $form[0].reset(); // clear the comment field before hiding the form for a fresh start
      $form.hide();
  });
```

Reply:
Ahmhá, yeah, easier access to the parent form, thank you. But `.reset()` is not needed, it's needlessly destructive, and it doesn't ask confirm before destruction, being the only warning is the passive protection `.is-danger`.

Reply:
However, it's what users expect a Cancel button to do - if you click Cancel, you _want_ to discard whatever you had there. Not doing so subverts those expectations.

Reply:
"Cancel" doesn't allow newcomer to expect (or not to expect) a confirmation modal, nor does convey any definitive information about the content being discarded. I'm keen to apply the reset suggestion with the "Discard" suggestion.